stores/amazon_monitoring.py

```diff
@@ -95,6 +81,7 @@
 
 PDP_PATH = "/dp/"
 REALTIME_INVENTORY_PATH = "gp/aod/ajax?asin="
+BAD_PROXIES_PATH = "html_saves/bad_proxies.json"
```

Comment:
Given that html_saves can sometimes have tens or hundreds of thousands of files in it, I think it would be better to put this in `config` so it doesn't get lost or accidentally deleted

Reply:
done